live-examples/html-examples/table-content/caption.html

```diff
@@ -0,0 +1,28 @@
+<table>
+    <caption>He Man and Skeletor facts</caption>
+    <tr>
+        <td>&nbsp;</td>
+        <th scope="col">He Man</th>
+        <th scope="col">Skeletor</th>
+    </tr>
+    <tr>
+        <th scope="row">Role</th>
+        <td>Hero</td>
+        <td>Villain</td>
+    </tr>
+    <tr>
+        <th scope="row">Race</th>
+        <td>Human</td>
+        <td>Gar</td>
+    </tr>
+    <tr>
+        <th scope="row">Weapon</th>
+        <td>Power Sword</td>
+        <td>Havoc Staff</td>
+    </tr>
+    <tr>
+        <th scope="row">Dark secret</th>
+        <td>Expert florist</td>
+        <td>Cries at romcoms</td>
+    </tr>
```

Comment:
I think we should omit one of these rows (but not "Dark secret", we should keep that one).

As it is, at narrower widths the caption is partly outside its container.